discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -12,17 +12,20 @@
 from test_infra.consts import resources
 from test_infra.controllers.node_controllers.libvirt_controller import LibvirtController
 from test_infra.controllers.node_controllers.node import Node
-from test_infra.helper_classes.config import BaseTerraformConfig
+from test_infra.helper_classes.config import BaseTerraformConfig, BaseClusterConfig
```

Comment:
why we need BaseTerraformConfig and BaseClusterConfig ? whats the point to have base cluster if only one config of such class will be implemented ? what i miss?

Reply:
I think BaseTerraformConfig should be NodeControllerConfig or something like that

Reply:
OK, I'll open another task for that